### PR TITLE
Implement sorting command for player based on height,weight,jersey

### LINKED ISSUE #158: As a user, I want to sort the players in ascending/ descending order of weight/height/jerseynumber

### PR DESCRIPTION
**Pull request by DALIN-Prog**
Currently there is no option for user to sort players based on criteria. 
Lets implement a sorting command to allow users to sort based on height, weight, jersey number in both ascending and descending order.

Note
Error message still needs to 'fine tune'
**Scrutinisation is required as the implementation edits AB3, UniquePersonList refresh method**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            + PREFIX_JERSEY_NUMBER + "SORTING_ORDER\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_PLAYER + " " + PREFIX_HEIGHT + "asc";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": sort players based on the specified criteria.\n"
+            + MESSAGE_USAGE_PLAYER;
+
+    private final Prefix itemList;
+    private final Prefix sortingCriteria;
+    private final String sortingOrder;
+
+    public SortCommand(Prefix itemList, Prefix sortingCriteria, String sortingOrder) {
```
Comment (FYimu) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:33]: If sort command can only be used to sort player, is prefix P/ still necessary?

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        return null;
+    }
+
+    private void performSort(Model model) {
+        if (sortingCriteria.equals(PREFIX_HEIGHT)) {
+            if (sortingOrder.equals("asc")) {
+                model.sortPersonsInMyGM(Comparator.comparing(Person::getHeight));
+            } else {
+                model.sortPersonsInMyGM(
+                        (Person person1, Person person2) -> person2.getHeight().compareTo(person1.getHeight()));
```
Comment (FYimu) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:55]: Same as I envisioned for now, Iwill try to simplify this in the v1.4

**Review by FYimu — APPROVED** [review]
Good job on implementing a new command!

File: src/main/java/seedu/address/logic/commands/SortCommand.java (hunk 1 above)
Comment (DALIN-Prog) [reply src/main/java/seedu/address/logic/commands/SortCommand.java:33]: True that. In that case I will remove the prefix P/

**Pull request merged**